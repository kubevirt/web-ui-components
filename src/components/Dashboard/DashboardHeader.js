import React from 'react';
import PropTypes from 'prop-types';

const DashboardHeader = ({ children }) => (
  <div>
    <h1>{children}</h1>
  </div>
);

DashboardHeader.defaultProps = {
  children: null,
};

DashboardHeader.propTypes = {
  children: PropTypes.node,
};

export default DashboardHeader;
