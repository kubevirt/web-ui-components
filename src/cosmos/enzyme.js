import createContext from 'react-cosmos-test/enzyme';

import paths from '../../config/paths';

export default function createTestContext(args) {
  args.proxies = args.proxies || []; // workaround of react-cosmos-test bug
  return createContext({
    ...args,
    cosmosConfigPath: `${paths.config}/cosmos.config.js`,
  });
}
