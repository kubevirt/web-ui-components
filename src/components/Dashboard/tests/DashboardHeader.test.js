import React from 'react';
import { shallow } from 'enzyme';

import DashboardHeader from '../DashboardHeader';

const testDashboardHeader = () => <DashboardHeader>content</DashboardHeader>;

describe('<DashboardHeader />', () => {
  it('renders correctly', () => {
    const component = shallow(testDashboardHeader());
    expect(component).toMatchSnapshot();
  });
});
