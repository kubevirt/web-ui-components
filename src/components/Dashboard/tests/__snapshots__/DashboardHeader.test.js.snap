// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardHeader /> renders correctly 1`] = `
<div>
  <h1>
    content
  </h1>
</div>
`;
